utils/rowgroup/rowaggregation.h

```diff
@@ -630,6 +632,7 @@ class RowAggregation : public messageqcpp::Serializeable
   joblist::ResourceManager* fRm = nullptr;
   boost::shared_ptr<int64_t> fSessionMemLimit;
   std::unique_ptr<RGData> fCurRGData;
+  bool fRollupFlag = true;
```

Comment:
Could you elaborate why do you set this to true by default?

Reply:
I should not, of course. An artifact of the development process.